opm/autodiff/SimulatorBase_impl.hpp

```diff
@@ -111,10 +111,6 @@ namespace Opm
             adaptiveTimeStepping.reset( new AdaptiveTimeStepping( param_, terminal_output_ ) );
         }
 
-
-
-        output_writer_.writeInit( geo_.simProps(grid_) , geo_.nonCartesianConnections( ) );
```

Comment:
Looks like you also discovered an extra call: looks to me like we currently always write init twice!